rero/rero-ils
# Documentation resources: circulation actions, reroils_resources and loan state chart

## Why are you opening this PR?

* To have an history about circulation actions documentation
* To update reroils_resources chart using `dot` from graphviz package
* To update Loan state chart using `dot` from graphviz package

## How to test?

* Read **doc/README.md** and check each file/directory
* Check **doc/circulation/README.md** on Github to see if any SVG file is displayed with Loan state diagram
* Use Makefile files:

```bash
cd doc
make clean && make
cd circulation
make clean && make
```

and check all generated *.png and *.svg files.

**NOTE**: in a next step I will create a little bit documentation about Graphviz to understand how it works in RERO and how to solve some problem in creating graphs.

## Code review check list
